app/AndroidManifest.xml

```diff
@@ -48,7 +48,12 @@
             android:name=".provider.CaseDataContentProvider"
             android:authorities="org.commcare.dalvik.case"
             android:enabled="true"
-            android:readPermission="org.commcare.dalvik.provider.cases.read" />

```

Comment:
Hey, so unfortunately we have a few people using the old flag out there already, so we can't really change the permission out from underneath them :/